nautobot/extras/migrations/0003_initial_part_3.py

```diff
@@ -69,6 +69,11 @@ class Migration(migrations.Migration):
             name="tags",
             field=models.ManyToManyField(blank=True, related_name="_configcontext_tags_+", to="extras.Tag"),
         ),
+        migrations.AddField(
+            model_name="configcontext",
+            name="dynamic_groups",
+            field=models.ManyToManyField(blank=True, related_name="_configcontext_dynamic_groups_+", to="extras.DynamicGroup"),
+        ),
```

Comment:
This needs to be a new migration file, not an addition to an existing one, in order to support upgrade/downgrade cases.